Dockerfile

```diff
@@ -0,0 +1,19 @@
+# pull official base image
+FROM node:13.12.0-alpine
+
+# set working directory
+WORKDIR /app
+
+# add `/app/node_modules/.bin` to $PATH
+ENV PATH /app/node_modules/.bin:$PATH
+
+# install app dependencies
+COPY package.json ./
+COPY package-lock.json ./
+RUN npm install --silent
```

Comment:
I only recently learned that you should use [`npm ci`](https://docs.npmjs.com/cli/ci.html) to install dependencies for a production setup.